Apps/WebAPI/sdLitica.WebAPI/Controllers/v1/SampleEventController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using sdLitica.Events.Abstractions;
+using sdLitica.Events.Integration;
+using sdLitica.WebAPI.Entities.Common;
+
+namespace sdLitica.WebAPI.Controllers.v1
+{
+    /// <summary>
+    /// This controller is a sample conmtroller that requires authentication
+    /// </summary>
+    [Produces("application/json")]
+    [Route("api/v1/sampleevent")]
+    //[Authorize]
+    public class SampleEventController : BaseApiController
+    {
+        private readonly IEventBus _eventBus;
+
+        public SampleEventController(IEventBus eventBus)
+        {
+            _eventBus = eventBus;
+        }
+        /// <summary>
+        /// This REST API handler returns the list of domain projects
+        /// </summary>
+        /// <returns>List of domain projects</returns>
+        [HttpGet]
+        public async Task<NoContentResult> Get([FromQuery] PaginationProperties pagination)
```

Comment:
If we are posting something in the queue it is better to use POST/PUT methods. This is just an example, as far as I understand, so doesn't matter much, but just a remark for future

Reply:
Yes, this is just an example for the team about how to use the bus.